spec/models/topic_spec.rb

```diff
@@ -1,5 +1,40 @@
 require "rails_helper"
 
-RSpec.describe Topic, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+describe Topic do
+  describe "scopes" do
+
+     let(:public_topic) do
+       described_class.create(name:       "Topic Name",
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use 2 (not 1) spaces for indentation.